src/components/organisms/OrderPanel/index.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useEffect, useState } from 'react';
+import { getAllByParent } from 'api/firebase/firestore/firestore-actions';
+import { Category } from 'model/documents/category';
+import { Dish } from 'model/documents/dish';
+import { Flex } from '@chakra-ui/react';
+import { isEmpty } from 'others/helper-functions';
+import { Product } from 'model/documents/products';
+import { isSummaryDocument, OrderDocument } from 'model/order/order-types';
+import OrderBreadcrumb from 'components/molecules/OrderBreadcrumb';
+import OrderTilesGroup from 'components/atoms/OrderTilesGroup';
+
+type OrderPanelProps = {
+  addSummaryItem: (document: OrderDocument) => void;
+};
+
+const OrderPanel = ({ addSummaryItem }: OrderPanelProps) => {
+  const [breadcrumbState, setBreadcrumbState] = useState<Category[]>([]);
+  const [categories, setCategories] = useState<Category[]>([]);
+  const [dishes, setDishes] = useState<Dish[]>([]);
+  const [products, setProducts] = useState<Product[]>([]);
+
+  useEffect(() => {
+    getCategories();
+    getDishes();
+    getProducts();
+  }, [breadcrumbState]);
+
+  const getCategories = (): void => {
+    getAllByParent<Category>(
+      'categories',
+      'categories',
+      'parent',
+      isEmpty(breadcrumbState) ? undefined : breadcrumbState[breadcrumbState.length - 1].id,
+    )
+      .then((documents) => setCategories(documents))
+      .catch((error) => {
+        throw new Error(`Could not fetch categories!. Error: ${error.message}`);
+      });
+  };
+
+  const getDishes = (): void => {
+    if (!isEmpty(breadcrumbState))
+      getAllByParent<Dish>('dishes', 'categories', 'category', breadcrumbState[breadcrumbState.length - 1].id)
+        .then((documents) => setDishes(documents))
+        .catch((error) => {
+          throw new Error(`Could not fetch dishes!. Error: ${error.message}`);
+        });
+  };
+
+  const getProducts = (): void => {
+    if (!isEmpty(breadcrumbState))
+      getAllByParent<Product>('products', 'categories', 'category', breadcrumbState[breadcrumbState.length - 1].id)
+        .then((documents) => setProducts(documents))
+        .catch((error) => {
+          throw new Error(`Could not fetch products!. Error: ${error.message}`);
+        });
+  };
+
+  const onDocumentClick = (document: OrderDocument) =>
+    isSummaryDocument(document)
+      ? addSummaryItem(document)
+      : setBreadcrumbState((previousState) => [...previousState, document]);
+
+  const changeBreadcrumbState = (category: Category) => {
+    const foundIndex = breadcrumbState.findIndex((stateCategory) => stateCategory.id === category.id);
+    setBreadcrumbState((previousState) =>
+      previousState.filter((_, stateCategoryIndex) => stateCategoryIndex <= foundIndex),
+    );
+  };
+
+  const resetBreadcrumbState = () => {
+    setBreadcrumbState([]);
+    setDishes([]);
```

Comment:
I haven't thought about that during implementation.
We discussed that it would require some more time and effort to implement at this stage.
I'll create separate task in Jira and come back for that when we will be merging order panel, tables made by @kacpergadula and final screen into one entity.